bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -799,6 +723,42 @@ impl Client {
             });
         }
     }
+
+    fn retrieve_session(&self) -> Result<SessionTokens, String> {
```

Comment:
`String` seems strictly worse than `anyhow::Error`, any reason to use it here?

Reply:
Good point, I've enabled anyhow conditionally on OIDC, in the sdk crate.